ibis/backends/tests/test_temporal.py

```diff
@@ -306,6 +307,29 @@ def test_temporal_binop(backend, con, alltypes, df, expr_fn, expected_fn):
     backend.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    ('timedelta', 'temporal_fn'),
+    itertools.product(
+        ['5W', '3d', '2h', '3m', '10s'],
```

Comment:
Please also test really large timedelta like 100 years to make sure we don't overflow

Reply:
Please add also test for unsupported shifts (e.g. 1 nanos)

Reply:
Added test for both